examples/demo-app/src/app.tsx

```diff
@@ -194,6 +194,76 @@ const App = props => {
     );
   }, [dispatch]);
 
+  const _loadVectorTileData = useCallback(() => {
```

Comment:
Nit: Maybe adopt some naming convention for wrapped callbacks? 
```suggestion
  const handleLoadVectorTileData = useCallback(() => {
  const loadVectorTileDataCallback = useCallback(() => {
```